LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Remy Sharp, https://remysharp.com <remy@remysharp.com>
```

Comment:
Hey @markokajzer, if you can update this line to read:

```suggestion
Copyright (c) [project-start-date] - present Remy Sharp, https://remysharp.com <remy@remysharp.com>
```

I am sure @remy would be happy to merge this. @remy what would be an accurate year to use for the project start date?

Reply:
Of course, we can do that. Seems like the project was established in 2010. Would that work?